9.extension/README.md

```diff
@@ -0,0 +1,158 @@
+# Extending Dapr
+ 
+This sample explores the extendability of Dapr. There are use cases where you want to add custom functionality to Dapr, for example, you have an existing library logic like `Encryption/Decryption` that you would like to expose as an endpoint along with the Dapr API. In this sample you'll be creating a custom component `Crypto` and add it to the Daprd binary as a gRPC endpoint.
```

Comment:
Proposed:

This sample demonstrates the extensibility of Dapr. Your organization may have a requirement to add custom functionality to Dapr, such as integration with propriety or legacy systems.  Custom Components is a point of extension that allows third-party functionality to run alongside the existing building blocks that ship with Dapr.

For example, you have an existing library for encryption and decryption that you would like to expose as a building block to your applications. In this sample you'll be creating a custom component called `Crypto` and adding it to the `daprd` binary. Applications will connect to this component via a gRPC endpoint which is the same mechanism the Dapr SDKs uses.